migrations/versions/4e75c0ed029e_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
```

Comment:
this can be deleted since it's empty. If you already upgraded, deleting it might cause issues for you. Just do `python manage.py db downgrade` to roll it back to the previous version